dazel/README.md

```diff
@@ -45,30 +42,30 @@ You can `bazel run` files in `bin/`:
 $ bazel run :hello_bin
 ```
 
-You can also `bazel run` a development sever for your web application:
+You can also run a development sever for your web application:
 
 ```bash
 # Assume you have web/main.dart, and web/index.html.
-$ bazel run :main_dartium_serve
+$ pub run dazel serve :main_dartium_serve
```

Comment:
Updated to reflect that both Dart and js are served with a single command. I can't quite remember the details on how to hit the right URL for each so we should update the documentation. Since this isn't a regression in this change I'll punt on that for now.